packages/web-components/src/components/back-to-top/back-to-top.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, property, customElement, LitElement } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import settings from 'carbon-components/es/globals/js/settings.js';
+import UpToTop20 from 'carbon-web-components/es/icons/up-to-top/20.js';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+// import './back-to-top-button';
+import styles from './back-to-top.scss';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Back to top
+ *
+ * @element dds-back-to-top
+ */
+@customElement(`${ddsPrefix}-back-to-top`)
+class DDSBackToTop extends StableSelectorMixin(LitElement) {
+  /**
+   * The document height
+   */
+  private _bodyHeight: any | null = null;
+
+  /**
+   * The viewport height
+   */
+  private _windowHeight: any | null = null;
+
+  /**
+   * Button click scrolls to top
+   */
+  // eslint-disable-next-line class-methods-use-this
+  private _handleOnClick() {
+    window.scrollTo({ top: 0, behavior: 'smooth' });
```

Comment:
Instead of using global `window`, we make it clear the relationship between the component DOM and the window object:

```suggestion
    this.ownerDocument!.defaultView!.scrollTo({ top: 0, behavior: 'smooth' });
```